public/app/core/components/QueryOperationRow/QueryOperationRow.tsx

```diff
@@ -154,7 +154,6 @@ const getQueryOperationRowStyles = stylesFactory((theme: GrafanaTheme) => {
       align-items: center;
       flex-grow: 1;
       cursor: pointer;
-      overflow: hidden;
```

Comment:
this breaks other uses of this component

Reply:
like QueryEditors and TransformationRows & QueryOptions

Reply:
Thanks for pointing out. The only other way I could think of was to set `flex-shrink: 100 /* or some other large value */` on [DetailText](https://github.com/grafana/grafana/blob/4b150297c0eb3484fa40c977f81b818d04fb1e49/public/app/features/inspector/DetailText.tsx#L15). Let me know if thats okay or you have any other suggestions.

Reply:
hmm, do you really need to remove the "overflow:hidden" in the solution without the scrolling?

Reply:
With "overflow:hidden" the title wrapper shrinks and hides the text.

![image](https://user-images.githubusercontent.com/76885189/115897764-65740600-a47a-11eb-94b4-00a1556b40a3.png)


Removing overflow breaks other uses of the component. flex-shrink on the DetailText element was the only alternative I could come up with, which requires minimal changes.